src/main/java/com/gmail/yuramitryahin/util/ListSupplier.java

```diff
@@ -0,0 +1,16 @@
+package com.gmail.yuramitryahin.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ListSupplier {
+    private static final int LENGTH = 1000000;
```

Comment:
```suggestion
    private static final int LENGTH = 1000_000;
```
it's a good practice to make large numbers more readable

Reply:
fixed